index.html

```diff
@@ -213,10 +180,18 @@ <h6 class="title is-5">
 
             </div>
           </div>
-          <div class="select ml-5">
+          <div class="select ml-5 level-item">
             <select id="citySelector"></select>
           </div>
         </div>
+
+        <div class="level-item buttons is-right" style = "text-align: right;">
+          <button class = 'button is-text' onclick = "window.open('disclaimer.html','_blank')" target="_blank" rel="noopener noreferrer">Our disclaimer</button>
+        </div>
+        
```

Comment:
Remove the empty lines.

Reply:
Hey @tsrivishnu Do you want me to remove the extra lines (when there are more than one) wherever they occur in index.html (even in parts I didn't edit)?